mcux/README

```diff
@@ -91,9 +91,12 @@ Patch List:
   - Patched mcux-sdk\components\conn_fwloader\mcux-sdk\components\conn_fwloader\driver_conn_fwloader.cmake for TFM support
   - mcux-sdk/drivers/s3mu/fsl_s3mu.c: rename _BIT macro to avoid collision from util_macro.h in Zephyr
   - Modifed mcux\mcux-sdk\drivers\mipi_dsi_split\fsl_mipi_dsi.c line 541 to force 8.0 to float (8.0f) to avoid [-Double-promotion] warning
-  - Wrap conditional code setting SCB->VTOR with !defined(__ZEPHYR__) in system c files of the folloing parts: 
+  - Wrap conditional code setting SCB->VTOR with !defined(__ZEPHYR__) in system c files of the folloing parts:
```

Comment:
I guess we don't have check-compliance on the HAL... looks like you trimmed spaces. Can you also fix the "folloing" to be "following"?

Reply:
interesting, i didnt even realize i did that, i put an autocommand on my editor to remove the spaces at ends of lines when i save the file